#! /bin/sh /usr/share/dpatch/dpatch-run
## 07_typos.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: s/persistant/persistent/, s/Januar/January/ in man page.
## DP: Several typos in user-manual fixed.

@DPATCH@
diff -urNad privoxy~/doc/source/privoxy-man-page.sgml privoxy/doc/source/privoxy-man-page.sgml
--- privoxy~/doc/source/privoxy-man-page.sgml
+++ privoxy/doc/source/privoxy-man-page.sgml
@@ -367,7 +367,7 @@
  
  # Set personal exceptions to the policies in default.action #######
 
- # Sites where we want persistant cookies, so allow *all* cookies
+ # Sites where we want persistent cookies, so allow *all* cookies
  {-crunch-cookies -session-cookies-only}
   .redhat.com
   .sun.com
